mediaengine.go

```diff
@@ -78,30 +78,25 @@ func (m *MediaEngine) PopulateFromSDP(sd SessionDescription) error {
 			}
 
 			var codec *RTPCodec
-			clockRate := payloadCodec.ClockRate
-			parameters := payloadCodec.Fmtp
-			codecName := strings.ToUpper(payloadCodec.Name)
-			switch codecName {
-			case PCMU:
-				codec = NewRTPPCMUCodec(payloadType, clockRate)
-			case PCMA:
-				codec = NewRTPPCMACodec(payloadType, clockRate)
-			case G722:
-				codec = NewRTPG722Codec(payloadType, clockRate)
-			case Opus:
-				codec = NewRTPOpusCodec(payloadType, clockRate)
-			case VP8:
-				codec = NewRTPVP8Codec(payloadType, clockRate)
-			case VP9:
-				codec = NewRTPVP9Codec(payloadType, clockRate)
-			case H264:
-				codec = NewRTPH264Codec(payloadType, clockRate)
+			switch {
+			case strings.EqualFold(payloadCodec.Name, PCMU) || strings.EqualFold(payloadCodec.Name, PCMA):
+				codec = NewRTPPCMUCodec(payloadType, payloadCodec.ClockRate)
```

Comment:
https://github.com/pion/webrtc/blob/f4bbe5d76f46142f0a5563d96cec21374fe29dc9/mediaengine.go#L155-L177
`RTPCodec.Name` and `RTPCodec.RTPCodecCapability.MimeType` fields are different between PCMU and PCMA.

Reply:
Oops my mistake, nice catch :/